test/Microsoft.ML.Benchmarks/CacheDataViewBench.cs

```diff
@@ -59,17 +60,19 @@ public void Setup()
             for (int i = 0; i < _positions.Length; ++i)
                 _positions[i] = rand.Next(Length);
 
-            _col = _cacheDataView.Schema.GetColumnOrNull("A").Value;
+            _col = _cacheDataView.Schema["A"];
             _seeker = ((IRowSeekable)_cacheDataView).GetSeeker(colIndex => colIndex == _col.Index);
             _seekerGetter = _seeker.GetGetter<int>(_col.Index);
         }
 
         [Benchmark]
         public void CacheWithCursor()
         {
+            var cols = new List<Schema.Column>();
```

Comment:
>cols [](start = 16, length = 4)

Why we create collection for one element?

Reply:
leftover from before the utilities. thanks for the catch, i'll review it all again. 

---
In reply to: [247208960](https://github.com/dotnet/machinelearning/pull/1975#discussion_r247208960) [](ancestors = 247208960)